bin/update_or_create_webhooks.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import json
+
+import click
+from openedx_webhooks.lib.github.utils import update_or_create_webhook
+
+click.disable_unicode_literals_warning = True
+
+
+@click.command()
+@click.argument('config-json', type=click.File('rb'))
+def cli(config_json):
+    """
+    Update or create the specified webhook into repos using a
+    configuration JSON file:
+
+    \b
```

Comment:
These are runtime arguments for the CLI, we could inline the arguments, but they're too complex to easily express that way.

We could store the webhook specs in `edx/repo-tools-data`. Let's discuss what you have in mind so I can understand your viewpoint better.